### PR TITLE
monitoring: Improve possible solutions for repo-updater alert

### PR DESCRIPTION
**Pull request by ryanslade**
Closes: https://github.com/sourcegraph/sourcegraph/issues/15376

### CODE REVIEW

File: doc/admin/observability/alert_solutions.md [hunk 1]
```diff
@@ -2240,7 +2240,13 @@ To learn more about Sourcegraph's alerting, see [our alerting documentation](htt
 
 **Possible solutions:**
 
-- Make sure there are external services added with valid tokens
+- An alert here indicates that no code host connections have synced in at least 9h0m0s. This indicates that there could be a configuration issue
+with your code hosts connections or networking issues affecting communication with your code hosts.
```
Comment (unknwon) [inline doc/admin/observability/alert_solutions.md:2244]: Disclaimer: I'm not a native speaker :D 

```suggestion
with your code host connections or networking issues affecting communication with your code hosts.
```

File: doc/admin/observability/alert_solutions.md [hunk 2]
```diff
@@ -2240,7 +2240,13 @@ To learn more about Sourcegraph's alerting, see [our alerting documentation](htt
 
 **Possible solutions:**
 
-- Make sure there are external services added with valid tokens
+- An alert here indicates that no code host connections have synced in at least 9h0m0s. This indicates that there could be a configuration issue
```
Comment (unknwon) [inline doc/admin/observability/alert_solutions.md:2243]: qq: Is it "no code host" or "at least one code host"?

**Review by unknwon — APPROVED** [review]
Thanks for adding these!

File: doc/admin/observability/alert_solutions.md (hunk 2 above)
Comment (ryanslade) [reply doc/admin/observability/alert_solutions.md:2243]: No code host. The alert fires if we haven't seen ANY sync for 9 hours.

**Pull request merged**